README.md

```diff
@@ -414,3 +414,30 @@ Note: skip this step and jump to next section if you already have the compiled `
 
 * Run the assemble task by selecting `android-cmp-app:cmplibrary [assemble]` (should be already selected) and clicking the build icon (or selecting Build > Make Project) from the menus.
 * The release version of the compiled binary should be under `cmplibrary/build/outputs/aar/cmplibrary-release.aar` directory. Copy this file and import it to your project using the steps below.
+
+# Artifact Release Process
+## Release
+To publish new release artifact you need to do following:
+* Checkout `develop` branch and pull the latest commits.
+* Create a new branch based on pure develop branch code named `release/x.y.z` where x.y.z stands for ordinal version of upcoming release. Push it to `origin`. 
+* Using your preffered text editor, go to `cmplibrary/grade.property` and upgrade `VERSION_NAME = x.y.z`. Don't forget to save the changes!
+* Then go to `cmplibrary/release_note.txt`, clear it and fulfill with description of every single commit pushed to `develop` branch since the last artifact release. Please, stick to style of description which appears in that file! This part will appear in changelog after artifact release will be accomplished.
+* Commit and push these two files **ONLY**. The commit message **must** be “release/x.y.z”
+
+**From now on, committing to `develop` branch is forbidden until successful artifact release.**		
```

Comment:
ok, this is the reason, can we edit this sentence to express it as the reason of the previous sentence